libraries/testing/tester.cpp

```diff
@@ -556,6 +557,23 @@ namespace sysio { namespace testing {
                                    .active   = active_auth,
                                 });
 
+      include_ram_gift &= has_roa;
+      include_ram_gift &= a.prefix() != config::system_account_name;
+      if (include_ram_gift) {
+         // if bios contract with setalimits available then provide ram for account which is similar to what sysio.system contract does
+         const auto& acnt = control->get_account(config::system_account_name);
+         auto abi = acnt.get_abi();
+         if (std::ranges::any_of(abi.actions, [](const auto& action) { return action.name == "setalimits"_n; })) {
+            trx.actions.emplace_back(get_action(config::system_account_name, "setalimits"_n,
+                                                vector<permission_level>{{config::system_account_name, config::active_name}},
+                                                fc::mutable_variant_object()
+                                                ("account", a)
+                                                ("ram_bytes", 2800) // matches ram_newaccount of system contract
```

Comment:
could we use an explicit constant for this value?  Ideally, one that's shared w/ the system contract, but if that's not feasible I'd still like to see it as some extremely explicit const.